mlflow/server/js/src/components/App.js

```diff
@@ -54,6 +55,7 @@ class App extends Component {
                 <Route exact path={Routes.compareRunPageRoute} component={CompareRunPage}/>
             </div>
           </AppErrorBoundary>
+          <Route component={PageNotFoundView}/>
```

Comment:
I think we may want to swap this line with the one above it, otherwise the PageNotFoundView always renders :)

Reply:
Hm, this wasn't the case for me, could you show me how to reproduce that?